### ISSUE TITLE
TUI: Advanced options/Bundle Selection should show selected bundles

### ISSUE THREAD

**Issue by mvincerx** [clearlinux/distribution#411]
Problem:
As of clear-27150-installer.img, in `Advanced options/Bundle Selection`, the TUI installer does not show the specific bundles the user selects after the user selects them. This is inconsistent with other submenus that do show configuration values after the user selects them (e.g., Proxy, User Manager, etc.) prior to installation.

Proposed solution:
In `Advanced options/Bundle Selection`, the submenu should show which bundles were selected for installation after the user has selected and confirmed them in the Bundle Selection submenu.

**Comment by mdhorn** [comment]
@mvincerx I just checked the latest version with the release clr-installer and I do not see this issue.

**Comment by mdhorn** [comment]
@mvincerx I also just switch to clear linux 27150 which has an older clr-installer (1.1.2), but it also works as expected.

If you can reproduce this, please provide the swupd info, and screen shots if possible.

**Comment by mvincerx** [comment]
@mdhorn, thanks for the response. I'll test this when I'm back in the office on Monday. WFH now. Thanks.

**Comment by mdhorn** [comment]
@mvincerx Were you able to test this? I can't reproduce this.

**Comment by gvancuts** [comment]
Side question: how is the list of bundles to choose from created? It is not an exhaustive list of all and every bundle available so I suspect it's either filtered somewhere or hard-coded in the installer code?

**Comment by mdhorn** [comment]
@gvancuts The possible bundle choices during an interactive installation is a curated list kept in a configuration file that the installer uses. This allows people building their own installer images to easily configure the possible bundles to select during install. Manual installations via a command line have the option of specifying any valid bundle.

**Comment by gvancuts** [comment]
> @gvancuts The possible bundle choices during an interactive installation is a curated list kept in a configuration file that the installer uses. This allows people building their own installer images to easily configure the possible bundles to select during install. Manual installations via a command line have the option of specifying any valid bundle.

Thanks for the clarification!

**Comment by mdhorn** [comment]
@mvincerx Where you able to reproduce this?  Or can we close this issue?

**Comment by mdhorn** [comment]
@mvincerx Where you able to reproduce this? Or can we close this issue?

**Comment by mvincerx** [comment]
Yes, I reproduced the problem. Suppose, in this case, I've followed this order: 

1. I completed all `Required options`. 
2. I want to assign just two `Advanced options`.
2. I assigned a new user, `user123`,  in `User Manager`.
3. I want to select two bundles, `desktop autostart` and `editors` in `Bundle Selection`.

The expectation established (though unwritten) per TUI design is that *after* I  select and confirm values for a submenu, generally, the value(s) are reflected in the main menu (whether `Advanced options` or `Required options`).  

To show what contradicts user expectation, I added and confirmed `user123` in User Manager.   After I'm returned to the Main Menu, the TUI shows that value: `user123`. However, in the case of bundles (screenshot), after I selected bundles and returned to the Main Menu, it **does not show the bundles that I selected**; instead, it shows the same array of default bundle values as before. Therefore, as a user, I don't see visual confirmation that the values that I selected were assigned. 
 
FYI: I'm not sure whether this inconsistency was a result of lack of space in the Main Menu. It obviously would be difficult to show all bundles selected if a user selected more than 3 or 4. IMHO, if at least 1-3 bundles selected by the user were shown, the Bundle Selection submenu would appear consistent with all other submenus.

- 

![screenshot from 2019-02-28 18-32-36](https://user-images.githubusercontent.com/37549381/53589718-f2460200-3b44-11e9-85f0-23204055f021.png)
![screenshot from 2019-02-28 18-32-17](https://user-images.githubusercontent.com/37549381/53589721-f540f280-3b44-11e9-9ac1-aa212d5ebf6b.png)
